%!TEX root = ../thesis.tex
%*******************************************************************************
%****************************** Second Chapter *********************************
%*******************************************************************************

\chapter{The spatiotemporal expression of RdDM components and small RNA landscapes in the germline of \textit{Arabidopsis thaliana}}

\ifpdf
    \graphicspath{{Chapter2/Figs/Raster/}{Chapter2/Figs/PDF/}{Chapter2/Figs/}}
\else
    \graphicspath{{Chapter2/Figs/Vector/}{Chapter2/Figs/}}
\fi


\section{Abstract}


%\begin{figure}[htbp!] 
%\centering    
%\includegraphics[width=1.0\textwidth]{minion}
%\caption[Minion]{This is just a long figure caption for the minion in Despicable Me from Pixar}
%\label{fig:minion}
%\end{figure}

\section{Introduction}

\begin{itemize}
    \item Mae meiocytes are enclosed by a layer of somatic nurse cells
\end{itemize}

\section{Summary points}

CLSY regulated loci become less abundant in MS and SC SN. 
Genic features - more similar to somatic tissues.
Perfect matching HyperTE 24nt sRNA?



\section{NRPD1 is expressed in the tapetum, microspores but not in meiocytes}

Since RNA polymerase IV and V preferentially associate with methylated RNA, RdDM is thought of as a self-reinforcing pathway. Therefore the observation that there are few perfect matching 24nt sRNAs in MetGenes in Arabisopsis meiocytes, it was hypothesised that the Pol IV dependent pathway is quiescent and that the sRNAs directing DNA methyaltion at MetGenes are produced from HyperTE loci in the tapetum that target HyperTE and MetGene methylation in the meiocyte \citep{RN199,RN187}. To confirm this expression pattern, pNRPD1::NRPD1-eGFP and pNRPE1::NRPE1-eGFP fluorescent reporter lines (of the catalytic subunits of RNA Polymerase IV and V respectively) were constructed by dr Jincheng Long.

The columella in the root tip of Arabidopsis is hypermetylated in the CHH context \citep{RN261}, the root tip was imaged to screen and select the best lines selected for further imaging (Figure\ref{fig:Pol4_5_roots}).

\begin{figure}[htbp!] 
\centering    
    \includegraphics[width=1\textwidth]{Chapter2/Figs/Figure3_Pol4_anther_gynecium.pdf}
\caption{\textbf{NRPD1 is expressed in the tapetum, microspore and gynoecium.}}
\label{fig:Pol4_anther}
\captionsetup{font=small}
    \caption*{NRPD1 expression in meiocyte stage anthers in the tapetum and surrounding somatic cells(first row, third column (GFP, green, white arrows), and expression in microspores (second row third column, white arrows). NRPD1 is also expressed in both the germ and somatic cells of the meiocyte stage gynoecium (third row (GFP, green)). Scale bar 10 $\mu$m.}
\end{figure}

Meiocytes are surrounded by the tapetum - a somatic tissue layer of nurse cells with cytoplasmic connections to meiocytes through plasmodesmata. As expected, NRDP1 expression was confined to the tapetal layer and surrounding somatic anther tissue in meiocyte stage anthers (Figure \ref{fig:Pol4_anther}). Once meiocytes undergo meiosis, the plasmodesmatal connection to the tapetum ceases and the meiocytes are surrounded by callose cell walls. At the microspore stage, the germline cells express NRPD1 (Figure \ref{fig:Pol4_anther} second row, \ref{fig:Pol4_germ} first and second rows). Following a further two cell divisions, the bicellular and tricellular pollen grains are produced, neither of which express NRPD1 (Figure \ref{fig:Pol4_germ}, rows 3 and 4). However, it must be added that the expression levels of the best NRPD1 reporter lines were a lot lower and more diffuse than the expression of NRPE1 or CLSY3 reporter lines (Figures \ref{fig:CLSY3_anther}, \ref{fig:Pol4_5_roots}). Further, expression in germline tissues wasn't consistent even in lines that exhibited strong root signal (Figure \ref{fig:Pol4_germ}). As these results are from screening 37 independent transformants from the same transformation event, it would be worth constructing new transformants perhaps with a stronger fluorescent reporter (such as tdTomato) to corroborate these results, and confidently confirm whether NRPD1 is expressed in the pollen.

\begin{figure}[htbp!] 
\centering    
    \includegraphics[width=1\textwidth]{Chapter2/Figs/Figure4_Pol4_germline.pdf}
\caption{\textbf{NRPD1 is expressed in microspores, but its expression is absent in pollen}}
\label{fig:Pol4_germ}
\captionsetup{font=small}
    \caption*{NRPD1 expression early and late microsproes (first and second rows, third column (GFP, green, white arrows), and expression in microspores (second row third column, white arrows). NRPD1 however us not expressed in bicellular or tricellular pollen (third and fourth rows (DAPI, blue GFP, green). Scale bar 10 $\mu$m.}
\end{figure}

\section{NRPE1 is expressed in both somatic and germline tissues in \texit{Arabidopsis} anthers}

According to our hypothesis the PolV-dependent pathway of RdDM should be active in the somatic tapetal layer and meiocytes, 


\begin{figure}[htbp!] 
\centering    
    \includegraphics[width=1\textwidth]{Chapter2/Figs/Figure5_Pol5_germline.pdf}
\caption{\textbf{NRPE1 is expressed in the tapetum, meiocyte, microspores and gynoecium}}
\label{fig:Pol5_germ}
\captionsetup{font=small}
    \caption*{NRPE1 expression in meiocyte stage anthers in the tapetum, meiocyte and surrounding somatic cells(first row, third column (GFP, green, white arrows), and expression in microspores (second row third column, white arrows). NRPE1 is also expressed in both the germ and somatic cells of the meiocyte stage gynoecium (third row (GFP, green)). Scale bar 10 $\mu$m.}
\end{figure}


\section{CLSY3 is expressed in the tapetum of meiocyte stage anthers and in the gynoecium}
The hypothesis that the RNA Pol IV dependent part of the RdDM pathway is quiescent in meiocytes  dissect important key players 


\begin{figure}[htbp!] 
\centering    
    \includegraphics[width=1\textwidth]{Chapter2/Figs/Figure1_CLSY3_anther_gynecium.pdf}
\caption{\textbf{CLSY3 is expressed in the tapetum of meiocyte stage anthers and in the gynoecium}}
\label{fig:CLSY3_anther}
\captionsetup{font=small}
    \caption*{CLSY3 is strongly expressed in the nucleus of meiocyte stage tapetal cells (first row (Venus, yellow)), which then diminishes in the senescing tapetum after meiosis in the microspore stage(second row, Venus (yellow)). CLSY3 is also expressed in both the germ and somatic cells of the meiocyte stage gynoecium (third row (Venus, yellow)). Scale bar 10 $\mu$m.}
\end{figure}

\begin{figure}[htbp!] 
\centering    
    \includegraphics[width=1\textwidth]{Chapter2/Figs/Figure2_CLSY3_germline.pdf}
\caption{\textbf{CLSY3 is not expressed in the male germline cells}}
\label{fig:CLSY3_germ}
\captionsetup{font=small}
    \caption*{DAPI stained microspore, bicellular or tricellular pollen (second column (DAPI, blue). CLSY3 signal not detected in any germline cells (third row (Venus, yellow). Scale bar 10 $\mu$m}
\end{figure}


\section{Germline sRNA sequencing project}

\begin{figure}[htbp!] 
\centering    
    \includegraphics[width=1\textwidth]{Chapter2/Figs/Figure6_sRNA_sizes.pdf}
\caption{\textbf{Line graphs illustrating the size distribution of sRNAs in somatic and germline tissues}}
\label{fig:sRNA_sizes}
\captionsetup{font=small}
    \caption*{(A) The size distribution of sRNAs in leaf, root and seedling. X axis: sRNA lengths, Y axis: reads per million (RPM) (B) The size distribution of sRNAs in meiocyte, microspore, sperm cell, sperm nucleus and tapetum. X axis: sRNA lengths, Y axis: reads per million (RPM)}
\end{figure}

\begin{figure}[htbp!] 
\centering    
    \includegraphics[width=1\textwidth]{Chapter2/Figs/Figure7_Pie_charts.pdf}
\caption{\textbf{HyperTE derived sRNAs decline in proportional abundance after meiosis and throughout pollen development}}
\label{fig:sRNA_pie}
\captionsetup{font=small}
    \caption*{Pie chart showing the proportional mapping of 24nt sRNAs to genomic features of interest: HyperTEs, TEs, Genes, MetGenes and intergenic regions in (A) germline tissue (meiocyte, microspore, sperm cell and sperm nucleus) and (B) somatic tissue (seedling, root and leaf).}
\end{figure}

\begin{figure}[htbp!] 
\centering    
    \includegraphics[width=1\textwidth]{Chapter2/Figs/Figure8_Heatmaps_CLSY3_CLSY1_2vs_cRdDMs_HyperTEs.pdf}
\caption{\textbf{CLSY3-dependent loci and HyperTEs generate abundant sRNAs in meiocytes and tapetum, as well as in microspores and sperm cells, though to a lesser degree. CLSY1\&2-dependent loci and canonical RdDM loci primarily produce abundant sRNAs in somatic tissues, with intermediate levels in sperm cells.}}
\label{fig:hm_CLSY3_CLSY1}
\captionsetup{font=small}
    \caption*{Heatmaps showing the 24nt sRNA RPKM levels of (A) CLSY3 dependent, (B) CLSY1\&2 dependent, (C) HyperTEs and (D) cRdDM loci in different germline (meiocyte, tapetum, sperm cell, sperm nucleus, pollen) and somatic tissues (leaf, seedling, root).}
\end{figure}

\begin{figure}[htbp!] 
\centering    
    \includegraphics[width=1\textwidth]{Chapter2/Figs/Figure9_Scatterplots_HyperTEs_chromatin_SC.pdf}
\caption{\textbf{HyperTEs produce highly abundant 24nt sRNAs specifically in the meiocyte and tapetum which tapers off in the sperm cell. Loci with highly abundant 24nt sRNAs specifically in the sperm cell tend to be more heterochromatic.}}
\label{fig:scatter_SC_chromatin}
\captionsetup{font=small}
    \caption*{Correlations between 24nt sRNA abundance and the indicated cell types, highlighting (A-B) genomic features of interest and (C-D) chromatin structure. (C) Sperm cell specific heterochromatic cluster highlighed with red box. }
\end{figure}

\begin{figure}[htbp!] 
\centering    
    \includegraphics[width=1\textwidth]{Chapter2/Figs/Figure10_SC_hetero_clusters_methylation.pdf}
\caption{\textbf{Sperm-specific heterochromatic loci exhibit significantly lower overall methylation levels compared to the entire sperm dataset}}
\label{fig:SC_hetero_clusters}
\captionsetup{font=small}
    \caption*{CG, CHG and CHH methylation levels of all sperm clusters (dark green) and the heterochromatic 568 loci subset (light greeen). ***: p-value < 0.001, **: p-value < 0.01,*: p-value < 0.05}
\end{figure}

\begin{figure}[htbp!] 
\centering    
    \includegraphics[width=1\textwidth]{Chapter2/Figs/Figure11_Reactivated_MetGenes_heatmap.pdf}
\caption{\textbf{Approximately 20\% of MetGenes produce sRNAs in the sperm}}
\label{fig:hm_metgene_reactivated}
\captionsetup{font=small}
    \caption*{Clusters 1,2: sperm specific sRNA production. Cluster 3: HyperTE/MetGene boundary derived meiocyte and tapetum specific clusters}
\end{figure}

\begin{figure}[htbp!] 
\centering    
    \includegraphics[width=1\textwidth]{Chapter2/Figs/Figure12_Reactivated_MetGenes_methylation.pdf}
\caption{\textbf{The reactivated MetGenes gain CHH methylation in the sperm cell.}}
\label{fig:boxplot_MetGene_methylation}
\captionsetup{font=small}
    \caption*{(Box plots of CG (A), CHG (B) and CHH (C) methylation of all MetGenes compared to the reactivated subset, in germline and somatic tissues. Heatmaps of the CG (D) CHG (E) and CHH (F) methylation of sperm sRNA producing MetGenes. ***: p-value < 0.001, **: p-value < 0.01,*: p-value < 0.05}
\end{figure}

\begin{figure}[htbp!] 
\centering    
    \includegraphics[width=1\textwidth]{Chapter2/Figs/Figure13_TE_families_heatmap.pdf}
\caption{\textbf{ATGP2N TEs produce highly abundant 24nt sRNAs in the sperm cell, sperm nucleus and pollen}}
\label{fig:TE_families}
\captionsetup{font=small}
    \caption*{(A) Heatmap showing the 24nt sRNA production of TE families in germline and somatic tissues. cluster 9 contains the ATGP2N families. (B) Snapshot CHH methylation levels of an ATGP2N TE which produces abundant sRNAs in the sperm. (C) Boxplots of CHH methylation levels of the ATGP2 subset compared to all TEs in germline and somatic tissues. ***: p-value < 0.001, **: p-value < 0.01,*: p-value < 0.05}
\end{figure}


Overlap of 24 nucleotide sRNA clusters with genomic features of interest
Following on from work from last year, to further understand the sRNA landscape in sperm cells, the following analytic approach was used: Firstly, real 24 nucleotide sRNA clusters from 2 sperm cell replicates were filtered out (see Methods for filtering parameters), then the whole sperm cell cluster set was checked to see if any patterns could be identified relating to enrichment in CLASSY dependent loci, TE families and chromatin state, as well as some other genomic features of interest. Secondly, it was important to identify a subset of sperm cell specific loci (c513, see methods for this pipeline) and cross reference with the features of interest as well as the methylation levels in the subset.

Initially, it was important to get a snapshot of the distribution of sRNAs in different tissues within the full set of sperm cell specific clusters ( n = ~34000). In Figure 1 an annotation of TE families was examined and the RPKM value of the 24 nucleotide sRNAs at the full set of sperm clusters overlapping the TE annotation was plotted and as comparison, these loci were checked in other tissues. From this, we can see a highly derepressed set of TE families conserved between the pollen, sperm cell and sperm nucleus. This set of TE families was determined to be from the LTR/Gypsy superfamily, specifically ATGP2 and ATGP2N, the latter of which has previously been reported to gain chromatin accessibility and be transcriptionally upregulated in met1 mutants. In this study ATGP2N was part of a subset of 3 groups of TEs that gained chromatin accessibility and increased in transcription as well as an increase in the production of 21nt sRNAs in the inflorescences of met1 mutants. The other 2 groups gained accessibility but had no change in transcription or sRNA production, one already being highly expressed in wild type plants, and the other remaining repressed with respect to transcription and sRNA production even with increased chromatin accessibility 32.

Another group of TEs were also identified which had highest production of 24nt sRNAs specifically in the sperm cell, low sRNAs in pollen and intermediate sRNA levels in soma. These belonged to superfamilies LTR/Gypsy (ATGP1), DNA/HAT (ATHATN4, ATHATN7, SIMPLEHAT2) and DNA/MuDR (VANDAL2, VANDAL6).

Figure 2 depicts an updated diagram of the sRNA levels of HyperTEs overlapping sperm cell specific clusters in different tissues.

Here we can see that the HyperTEs overlapping these genomic regions are most highly expressed in the tapetum, meiocyte and microspore, with an intermediate expression in sperm cells and sperm nuclei. However, it must be highlighted that pollen shows higher relative levels of sRNAs when compared to the sperm cell, suggesting that the HyperTE derived sRNAs must be synthesised in the vegetative nucleus.

To further dissect the nature of sRNA clusters within sperm, Figure 3 shows the comparison of the logRPKM values of sperm cell specific sRNA clusters between meiocyte and sperm, highlighted by genomic features of interest. We can clearly observe here that a large subset of sperm specific clusters that are producing sRNAs overlap with HyperTEs and these are correspondingly very highly upregulated in the meiocyte.
On the other hand, loci in which RPKM values were low in the meiocyte but high in sperm were a mixture of canonical RdDM loci and other sperm specific loci that do not overlap any genomic features of interest.

Similarly, highlighting CLSY dependent loci in this same dataset yield results consistent with Figure 3. In Figure 4 loci which still are highly upregulated in the meiocyte overlap CLSY3 dependent loci, but surprisingly loci that produce sRNAs in sperm distinctly from the meiocyte do not exclusively overlap CLSY1 and CLSY2 dependent loci as would be expected in somatic tissues but rather a lot of these loci do not overlap any CLSY dependent loci at all. It has also been observed in Figure S2, that a small cluster (n = 18) of sperm and pollen specific loci have highly abundant sRNAs and overlap with CLSY3-4 dependent loci. This is not present in either CLSY single (CLSY3 or CLSY4) overlapping loci. These findings underline the hypothesis that sperm has a distinct sRNA profile from both earlier germline cells and soma.

Finally, checking whether loci which are synthesising abundant sRNAs in the sperm compared to meiocytes have different chromatin states with respect to H3K9me2 as a proxy for euchromatic or heterochromatic chromatin state, we can conclude that this subset tends more heterochromatic than loci that are still synthesising sRNAs in the meiocyte (Figure 5). However as described in the methods, the chromatin states in the annotation that was used was defined using somatic tissue, so this should be repeated with a H3K9me2 ChIP-Seq data from germline tissue.
 
When comparing sperm cell cluster derived 24 nucleotide sRNAs from sperm cells to the seedling, we can see a subset of loci that do not produce as much small RNA in soma as they do in sperm which almost all tend to be heterochromatic (Figure 6).

As expected, the full set of sperm cell specific clusters highly correlate between the sperm cell and sperm nucleus (Figure 7). Furthermore, loci which synthesise abundant sRNA (Figure 7 purple cluster where logRPKM > 6) in both sperm cell and sperm nucleus tend to be heterochromatic. 

To further confirm the identity of these sperm specific heterochromatic loci, another filtering step was implemented whereby clusters were filtered out that had less than 3 RPKM difference between the sperm cell and other somatic (root, seedling, and leaf) and other germline tissues (meiocyte and tapetum). This yielded 513 sperm and pollen specific loci (c513). When comparing this subset of loci in the sperm cell against the seedling (Figure 8), we can clearly identify that the majority (43\%) of loci were intermediate heterochromatic or heterochromatic. This dataset can be further grouped into 2 groups. Group 1 contained 66\% of loci that synthesise sRNAs in both seedling and sperm (seedling logRPKM > 0), of which 83\% were heterochromatic. Group 2 contains the remaining sRNA clusters, (seedling logRPKM = 0), of which 74\% do not overlap any H3K9me2 markers. It must be noted that these observations might be skewed as the H3K9me2 markers are derived from somatic tissue, therefore this analysis needs to be repeated with appropriate H3K9me2 data from the germline to draw any final conclusions however these data seem to suggest that sperm specific sRNA clusters which have low but non-zero sRNAs in the soma, tend to be heterochromatic (Figure 8 purple cluster).
 

Finally, it is important to check whether the c513 subcluster has a specific methylation pattern distinct from the whole set. We can conclude that generally, sperm methylation is consistently high, intermediate, and low in the CG CHG and CHH context throughout germline development respectively (Figure 9 A-C, red bars) (from meiocyte to sperm). This is in contrast with the methylation within the c513 subset which generally seems lower in sperm than the whole subset (Figure 9 bars labelled WT\_SC for wild type sperm cell). However, if we look at methylation in other tissues, we can immediately conclude that this pattern is not specific to sperm.

MetGene reactivation

It was observed in the previous report that a subset of MetGenes were producing sRNAs in the sperm cell, which is in stark contrast to other sex cells. Therefore, it was important to survey the methylation state of this subset compared to the whole set of MetGenes. Filtering the subset of MetGenes which were producing sRNAs, it was revealed that around 20\% of MetGenes produce sRNAs in sperm. Since previous work has identified that MetGenes produce no perfect matching sRNAs in the meiocyte and tapetum17, it is likely that the clusters which produce sRNAs in meiocyte and tapetum are either at HyperTE/MetGenes boundaries, or are HyperTE derived. However, the majority of reactivated MetGenes in the sperm cell and sperm nucleus do not overlap the same loci (Figure 10 clusters 1,2: sperm specific reactivation and cluster 3: HyperTE/boundary derived meiocyte and tapetum specific clusters). Furthermore, the subset of loci that are uniquely upregulated in sperm cell and sperm nuclei (Figure 10 clusters 1-2) are not enriched in CLSY dependent loci (only 5 are broadly CLSY dependent out of 53 sperm specific reactivated MetGenes).

When checking the methylation score of all sperm reactivated subset of MetGenes tend to have a higher methylation score in all germline tissues (methylation of MetGenes are low in soma in all sequence contexts), so the higher methylation is not sperm cell specific (Figure 11).




Summary

In conclusion, sperm has a distinct sRNA profile from both earlier germline cells and soma, identifying a cluster of TEs (ATGP2 and ATGP2N) and a cluster of CLSY3-4 dependent loci (n = 18) that are highly abundantly producing sRNAs specifically in the sperm cell and pollen respectively.

There is no significant unique population within any genomic features of interest in sperm specific clusters. A small cluster of highly conserved sperm specific small RNAs were identified, of which the majority are heterochromatic (though this analysis needs to be repeated with ChIP-Seq data from germline).

Whilst around 20\% of MetGenes are reactivated specifically in sperm cells and sperm nuclei, this does not result in overall higher methylation in these tissues in any sequence contexts.


\subsection{Overlap with features of interests (TE families, classy dependent loci, HyperTEs MetGenes)}

\subsection{H3K9me2 levels in soma and sperm}
Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.

\subsection{Methylation levels of sperm specific clusters and reactivated MetGenes}
Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.

\subsection{Phased small interfering sRNAs}
Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.


\section{Discussion}
Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.

\section{Materials and Methods}

\subsection{Plant materials and growth conditions}

\textit{Arabidopsis thaliana} plants were grown in 16h light/8h dark conditions, in 21°C, 70\% humidity in a controlled environment chamber on germination medium (GM) without glucose supplementation. The following lines have been grown: Col-0 wild type plant for the microspore and pollen sRNA sequencing libraries. For the anther imaging experiments, the transgenic reporter lines pCLSY1::CLSY1-eGFP (clsy1), pCLSY2::CLSY2-eGFP (WT), pCLSY3::-CLSY3-Venus (clsy3) and pCLSY4::CLSY4-eGFP (clsy4), pNRPD1::NRPD1-eGFP (WT), pNRPE1::NRPE1-eGFP (WT) were used with pAGO4::AGO4-Venus or ), pCLSY3::-CLSY3-Venus (clsy3) for a positive control.

\subsection{Microscopy}

Meiocyte and microspore stage anthers were dissected using a Leica dissecting microscope on 0.8\% agar and vacuum infiltrated in DAPI buffer (Galbraith’s buffer (45 mM MgCl2-6H2O, 30 mM trisodium citrate, 20 mM MOPS, pH 7.0), 1 µg/mL DAPI, and 0.01\% [v/v] Triton X-100) briefly, then imaged using imaging spacers (SecureSealTM Grace BioLabs). Individual microspores and pollen were imaged after isolation from unopened flower buds and opened flowers respectively, by collecting ~500 µL of flower buds into DAPI buffer, briefly vortexing and pelleting cells. All samples were imaged using Leica Stellaris 8 and Leica SP8X confocal microscopes.

\subsection{Isolation of microspores and pollen using fluorescence activated cell sorting}

\textit{Arabidopsis thaliana} microspores were isolated by manually collecting microspore stage flower buds, using the morphology described previously \citep{RN86}. The buds were collected into 1.5mL microcentrifuge tubes and suspended in PEB (10 mM CaCl2, 2 mM MES, 1 mM KCl, 1\% H3BO3, 10\% Sucrose, pH 7.5) buffer. The microspore cells were isolated as described previously \citep{RN140}. Briefly, the buds were gently ground in a clean pestle and mortar in PEB buffer and filtered through Miracloth (Merck-Millipore 475855) into a 1.5mL Eppendorf tube. This crude fraction was centrifuged for 5 minutes at 800g to gently pellet the microspore cells. The pellet was resuspended in 500µL PEB and sequentially filtered through 30µm and 20µm mesh filters (CellTrics®).

The fraction was separated using fluorescence-activated cell sorting (FACS) on a BD FACSMelody cell sorter (Beckton Dickinson) into TRI reagent (Zymo Research). The purity of the sorted cells was inspected using a widefield microscope.

Sperm cells were isolated as previously described \citep{RN140,RN141} except the sperm cell release step was repeated 4 times to maximise sperm cell release from pollen grain. Sperm cells were stained using SYBR green and SITOX orange and sperm cells and nuclei were sorted into TRI reagent (Zymo Research).

\subsection{sRNA sequencing}

RNA was released from isolated microspores by vortexing the samples with RNase free glass beads (Sigma-Aldrich) for 4 minutes before proceeding with RNA isolation. RNA was isolated from all samples using the Direct-zol™ RNA kit (Zymo Research, CN: R2061).

2 replicates of sRNA sequencing libraries were constructed per cell type (microspore, sperm cell, sperm nuclei) using the RealSeq®-biofluids NGS Library Preparation Kit for miRNAs and small RNAs. For the three library types the number of cells used for each replicate were as follows: 300 000 cells for microspore, 600 000 cells for sperm cells, and 1 500 000 cells for sperm nuclei.

Briefly, the RealSeq® adapters were ligated and blocked, then the sample was circularised, and the adapter dimers were removed. Then reverse transcription was performed as well as PCR amplification and size selection to end up with the final library.

Further, gel separation  and purification was performed to enrich the sRNA libraries for the desired fragment lengths (20-30 nucleotides long) as described previously \citep{RN187}. The libraries were ran on Novex TBE 6\% gels (Invitrogen, CN: EC6265BOX) along with a custom RNA ladder consisting of bands that were 146bp and 161bp in length. 

The gel was run for 45 mins with 120V and then stained using ethidium bromide. The gel was illuminated with UV light and the area between the two bands were then cut out using a clean razor blade and the gel slice was weighed. 2 volumes of elution buffer (0.5 M ammonium acetate, 10 mM magnesium sulfate, 1 mM EDTA (pH 8.0), 0.1\% SDS) was added to the gel slice. The sample was incubated at 37°C on a rotary platform (1000 RPM) for 4 hours. 

The sample was then centrifuged at 12000g for 1 minute at 4°C in a microcentrifuge. The supernatant was transferred into a fresh 1.5mL Eppendorf tube. An additional 200µL of elution buffer was added to the polyacrilamide gel fragment, vortexed, re-centrifuged and the two supernatants were combined. Any remaining fragments of polyacrilamide were removed  by passing the supernatant through a plastic column containng cellulose acetate filters. Two volumes of cold absolute ethanol was added and the solution was incubated on ice for 30 minutes. The DNA was recovered by centrifugation at 12000g got 10 minutes at 4°C. The pellet was dried and resuspended in 200µL TE buffer (10mM Tris-HCl, 1mM EDTA, pH 7.6) and 25µL of 3M sodium acetate (pH 5.2) was added. The precipitation step was repeated, the pellet was rinsed with 70\% ethanol and thoroughly dried before resuspending in TE buffer to a final volume of 30µL. The sample was incubated at 4°C overnight to redissolve the DNA.

The concentration and size distribution of purified sRNA libraries were determined using  High Sensitivity DNA Chips (Agilent Technologies, CN: 5067-5594).

The sRNA libraries were sequenced using a NextSeq 500 (Illumina).

\subsection{Bioinformatics and data analysis}

Adaptor trimming and filtering of reads that are 18nt to 28nt in length was performed on the raw sequencing data using Cutadapt \citep{RN88} v1.9.1). These reads were mapped to the Arabidopsis thaliana reference genome TAIR10 using Bowtie \citep{RN89}. 24nt clusters were extracted and loci were kept that overlapped previously determined non-overlapping genomic features of interest such as canocical RdDM loci (9629 loci), HyperTEs (797 loci), MetGenes (435 loci), transposons (TEs), genes17,20, and H3K9me2 enriched loci. These overlapping feature files were produced using BEDtools (v2.27.0) \citep{RN90}. 24nt sRNA clusters were extracted using ShortStack \citep{RN142} (n = 45639). To firstly identify true 24 nucleotide clusters, the ShortStack clusters from the two replicates were filtered to keep clusters where the predominant read length was 24 nucleotides long, where log2(RPKM+1) 24nt sRNA > log2(RPKM+1) 25nt sRNA +2 and that were >99bp (n = 34032).

Following this general filtering step, sperm cell specific clusters were identified where:
log2(RPKM+1) 24nt sperm cell sRNA > 0 \& where:
log2(RPKM+1) 24nt sperm cell sRNA - log2(RPKM+1) 24nt (other tissues: tapetum, meiocyte, microspore, root seedling and leaf) sRNA > 3 which resulted in 513 loci.

The euchromatic/heterochromatic annotation was allocated based on data from reference \cite{RN183} and classified into 5 chromatin states based on H3K9me2/H3: euchromatic (less than 0.6), intermediate euchromatic (bw. 0.6 and 0.9), intermediate (bw 0.9 and 1.4), intermediate heterochromatic (bw. 1.4 and 2.6), heterochromatic (above 2.6). 

The sRNA analysis pipeline is available from GitHub on request. Figures were plot using ggplot2 and pheatmap packages in R (v4.0.3).

