%!TEX root = ../thesis.tex
%*******************************************************************************
%****************************** Third Chapter **********************************
%*******************************************************************************
\chapter{The dynamic sRNA profile and inheritance of paternal epigenetic modifications post-fertilisation in the embryo of \textit{Marchantia polymorpha}}

% **************************** Define Graphics Path **************************
\ifpdf
    \graphicspath{{Chapter3/Figs/Raster/}{Chapter3/Figs/PDF/}{Chapter3/Figs/}}
\else
    \graphicspath{{Chapter3/Figs/Vector/}{Chapter3/Figs/}}
\fi


\section{Abstract}

\section{Introduction}

Recently, our group has shown that during spermatogenesis, there extensive DNA methylation reprogramming in Marchantia. As well as the deposition of 5-methylcytosine (5mC), there is an additional wave of DNA methylation reprogramming whereby N4-methylcytosine (4mC) - an epigenetic modification previously only found in prokaryotes – is deposited at the majority of CG cites across the genome \citep{RN189}. 

Data from mature embryo reveals that this 4mC methylation is not present. Several hypotheses could explain why this happens. The most likely is that following fertilisation, paternal epigenetic marks are lost passively through cell division. Alternatively, the marks could be maintained through the first few cell divisions and then are actively or passively lost. We also know that the pronuclei take 3 days to fuse \citep{RN139}, so it is also possible that removing paternal epigenetic marks is required for zygote formation. 

Therefore, my project here will help complement the work that has already been done on 4mC methylation in Marchantia; please see our lab’s preprint for other details of this project \citep{RN189}. Namely, I have already managed to isolate and image embryos 12 days after fertilisation (DAF), and I plan to isolate and stage embryos that are as young as possible to make sure that we can detect 4mC signals if paternal epigenetic marks are maintained through zygote formation but are passively lost during cell division.

\section{Overview of DNA methylation in the germline of Marchantia polymorpha}

Recently, our group has shown that during Marchantia spermiogenesis extensive DNA methylation reprogramming occurs. As well as the deposition of 5-methylcytosine (5mC), there is an additional wave of DNA methylation reprogramming whereby N4-methylcytosine (4mC) – an epigenetic modification previously only found in prokaryotes – is deposited at the majority of CG cites across the genome \citep{RN189}.

However, this methylation is absent in the mature embryo (containing over 1000 cells). The exact point in time when 4mC methylation disappears post-fertilisation remains unknown, as does whether the paternal genome retains any 4mC marks during the early stages of zygotic development. The first division of the zygote in Marchantia takes place around 4 days post-fertilisation1 which suggests that 4mC may be required to be actively removed for pronuclear fusion to occur. This aligns with the observed phenotype where embryos fertilised by Mpdn4mt1 sperm develop more rapidly, though this phenotype needs to be confirmed during early embryonic development. Alternatively, 4mC methylation could be lost passively following fertilisation through dilution by cell division, or actively removed post-fertilisation for DNA replication or cell division to function normally. 4mC may also be crucial in the correct genome dosage/transcriptional activity during early embryonic development. 

In Arabidopsis, 5mC methylation is lost actively to maintain homeostasis of DNA methylation levels through several DNA glycosylases, including ROS1 \citep{RN168}. Therefore, the orthologs of ROS1 in Marchantia, MpROS1a and MpROS1x (located on the female sex chromosome) offer possible candidates for erasing paternal 4mC methylation as they are expressed in the sperm \citep{RN212} and mature eggs/embryos \citep{RN257} respectively.

It has also recently been shown that other epigenetic marks, namely the heterochromatic histone modification H3K27me3 is deposited at the pronuclear stage exclusively on the paternal genome and causes the repression of the paternal genome persisting until the conclusion of embryogenesis \citep{RN160}. In contrast, in mammalian germlines, H3K27me3¬ mediated imprinting is limited to a few loci within the female gamete \citep{RN172}.

Hence, investigating the methylation profiles of early Marchantia embryos during the first zygotic divisions and examining the early developmental phenotype of Mpdn4mt1 mutants will complement our previous work, expanding our understanding of how paternal DNA methylation affects early embryonic development.


\section{sRNA analysis}
\subsection{Source of sRNAs in the sporophyte (paternal sRNAs with genome shut down?}
\subsection{Mismatch tageting \textit{de novo} methylation (SLMs)}

\section{AMD-seq of early Marchantia embryos}

\section{4mC/5mC methylation compared to H3K27me3 CutI\&Run}

\section{Developmental phenotype of Mpdn4mt1 mutant embryos}

\section{Discussion}

\section{Materials and Methods}

\subsection{Plant materials and growth conditions}

Male and female \textit{Marchantia polymorpha, L.} subsp. \textit{rudealis} accessions Takaragaike-1 (Tak-1, male) Takaragaike-2 (Tak-2, female) and Cam-2 was used. The plants were grown on 1\% agar (Sigma-Aldrich), supplemented with ½ strength Gamborg's B5 medium. They were grown in a controlled environment chamber (Conviron) at 21°C with 70\% humidity under constant light with far-red light for induction of sexual reproduction, as described previously \citep{RN212,RN254}.

\subsection{Plasmid construction and transformation}



\subsection{Microscopy} 

Tak-2 or Cam-2 archegonia were either fertilised with Tak-1 sperm or left unfertilised. The archegoniophores were collected for dissection and imaging and either observed live or fixed and stained for observation. For live cell imaging, the embryos were manually dissected, collected on cavity slides and washed twice with 1x PBS to remove any maternal tissue. They were then stained with DAPI for 5 mins and briefly vacuum infiltrated and observed using slides with imaging spacers (SecureSealTM Grace BioLabs) and/or cavities (BRAND® cavity slides) \citep{RN139}. 

For observing developmental synchronicity and the Mpdn4mt1 early developmental phenotype, rows of archegonia will be dissected from the base of digitate rays. Invoucres will be removed for clarity and samples will be fixed in PFA fixative (4\% formadehyde, 1x PBS, 0.01\% Triton X-100 and 1 ug/mL DAPI/Hoecht 3342 (ThermoFisher Scientific 62249), vacuum infiltrated 4 times for 2 minutes and incubating samples for 15 minutes on a gentle shake. Then the fixative will be discarded, and the sample washed twice using 1X PBS. The tissue will be cleared using ClearSee Alpha24 (50mL ClearSee25 + 0.315g sodium sulfite (Na2SO4) for a few days. The tissues will be mounted either using ClearSee Alpha or VECTASHIELD® and observed using imaging spacers (Gene Frame, ThermoFisher Scientific). The embryo/egg cells were imaged using directly using a Leica Stellaris 8 or Leica SP8X confocal microscope.


\subsection{AMD-Sequencing library construction and data analysis}

Dissected Tak-1 x Cam-2 embryos will be used to construct 4mC-AMD-seq libraries using ACCEL-NGS methyl seq DNA library kit (Qiagen, 59104) after treatment of sheared DNA with APOBEC (NEB \#E7120S), following manufacturer’s instructions.

Reads will be processed using SAMtools v1.9 \citep{RN174}, BEDtools v2.27.1 \citep{RN90} and Picard v2.18.27 \citep{RN173} then mapped to the Takv6 genome \citep{RN179} using bwa v0.7.17 \citep{RN182}. To distinguish between maternal and paternal sequencing reads, a SNP analysis will be conducted. SNPs will be called using gatk v4.0.1.2 \citep{RN177} wherein the SNPs between the parental Tak-1 and Cam-2 will be replaced with Ns. The parental reads will be assigned using SNPSplit v0.3.4 \citep{RN178} and counts for each parental genome will be calculated using SAMtools v1.9 and BEDTools v2.27.1. Please see reference \cite{RN160}  for a detailed description of the pipeline.
