%!TEX root = ../thesis.tex
% ******************************* Thesis Appendix B ********************************

\begin{figure}[htbp!] 
\centering    
    \includegraphics[width=1\textwidth]{Chapter3/Figs/Supps/FigureS1_Differences_in_development_embryos.pdf}
\caption{\textbf{\textit{Marchantia} embryos develop at different rates in different environmental conditions}}
\label{fig:embryo_diff}
\captionsetup{font=small}
    \caption*{DAPI stained \textit{Marchantia} embryos at 10 days after fertilisation (first row), and 12 days after fertilisation (second and third rows) showing different developmental stages. Scale bar 10 $\mu$m.}
\end{figure}

\begin{figure}[htbp!] 
\centering    
    \includegraphics[width=1\textwidth]{Chapter3/Figs/Supps/FigureS2_Fixing_MpSUN_trials.pdf}
\caption{\textbf{Fixing and staining of wild type (first row) ECPro::MpSUN-GFP embryos (third row) and gemmae (second row). Scale bar 10 $\mu$m.}}
\label{fig:MpSUN}
\captionsetup{font=small}
    \caption*{}
\end{figure}

\begin{figure}[htbp!] 
\centering    
    \includegraphics[width=1\textwidth]{Chapter3/Figs/Supps/FigureS3_enzyme_tests.pdf}
\caption{\textbf{Fixing and staining embryos after treatment with cell wall digesting enzymes}}
\label{fig:enzyme_tests}
\captionsetup{font=small}
    \caption*{Scale bar 10 $\mu$m.}
\end{figure}

\begin{figure}[htbp!] 
\centering    
    \includegraphics[width=1\textwidth]{Chapter3/Figs/Supps/FigureS_chrom_ridge_plot.pdf}
\caption{\textbf{}}
\label{fig:chrom_ridge}
\captionsetup{font=small}
    \caption*{}
\end{figure}

\begin{figure}[htbp!] 
\centering    
    \includegraphics[width=1\textwidth]{Chapter3/Figs/Supps/FigureS4_dividing_cells.pdf}
\caption{\textbf{4mC mutant embryos start dividing earlier than WT embryos}}
\label{fig:Dividing_cells}
\captionsetup{font=small}
    \caption*{Line graph that shows dividing cells in embryos fertilised by wild type (blue) or 2 independent 4mC mutant (green) sperm.}
\end{figure}

\begin{figure}[htbp!] 
\centering    
    \includegraphics[width=1\textwidth]{Chapter3/Figs/Supps/FigureS5_H3k9me1.pdf}
\caption{\textbf{H3K9me1 over genes and TEs is not associated with the paternal genome and tends maternal}}
\label{fig:h3k8me1}
\captionsetup{font=small}
    \caption*{}
\end{figure}

\begin{figure}[htbp!] 
\centering    
    \includegraphics[width=1\textwidth]{Chapter3/Figs/Supps/FigureS1_immunostaining.pdf}
\caption{\textbf{Immunostaining of H3K27me3 in 13 DAF embryos fertilised by wild type and dn4mt1 \#21 knockout sperm }}
\label{fig:immuno}
\captionsetup{font=small}
    \caption*{}
\end{figure}

\begin{table}[htbp]
    \centering
    \scriptsize % Keep the font size as scriptsize
    \caption{Primers used for nuclear reporter line cloning}
    \label{table:reporter_primers}
    \begin{tabular}{>{\raggedright\arraybackslash}p{3.4cm} >{\raggedright\arraybackslash}p{2.8cm} >{\raggedright\arraybackslash}p{4.7cm} >{\raggedright\arraybackslash}p{1.9cm}}
    \toprule
    Construct Target & Name & Sequence & Purpose \\
    \midrule
    pMpGWB115 or pMpGWB116 & attB1-fluo-NLS-attB2-f1 & \seqsplit{GGGGACAAGTTTGTACAAAAAAGCAGGCTTAATGGTGAGCAAGGGCGAG} & citrine/tdTom insert + attB sites \\
    pMpGWB115 or pMpGWB117 & attB1-fluo-NLS-attB2-r1 & \seqsplit{GGGGACCACTTTGTACAAGAAAGCTGGGTTCTATCCTCCAACCTTTCTCTTCTTCTTAGG} & citrine/tdTom insert + attB sites \\
    \midrule
    pMpGWB102-Citrine-NLS \& pMpGWB103-Citrine-NLS entry clones (pDONR221) & citrine-ent-seq-f1 & \seqsplit{TTAATGGTGAGCAAGGGC} & Entry clone sequencing \\
    & citrine-ent-seq-r1 & \seqsplit{TGGGTTCTATCCTCCAACC} & Entry clone sequencing \\
    pMpGWB102-tdTom-NLS \& pMpGWB103-tdTom-NLS entry clones (pDONR221) & tdTom-ent-seq-f1 & \seqsplit{AGGCTTAATGGTGAGCAAG} & Entry clone sequencing \\
    & tdTom-ent-seq-r1 & \seqsplit{GGGTTCTATCCTCCAACC} & Entry clone sequencing \\
    \midrule
    pMpGWB102-Citrine-NLS & 102-citrine-exp-seq-f1 & \seqsplit{TTGGAGAGAACACGGG} & Expression clone sequencing \\
    & 102-citrine-exp-seq-r1 & \seqsplit{CTGGGTTCTATCCTCCAAC} & Expression clone sequencing \\
    pMpGWB102-tdTom-NLS & 102-tdTom-exp-seq-f1 & \seqsplit{AGAGAACACGGGGGACTC} & Expression clone sequencing \\
    & 102-tdTom-exp-seq-r2 & \seqsplit{TCGGAGGAGGCGGTG} & Expression clone sequencing \\
    & 102-tdTom-exp-seq-r1 & \seqsplit{ACAAGAAAGCTGGGTTCTATCCTC} & Expression clone sequencing \\
    \midrule
    pMpGWB103-Citrine-NLS & 103-citrine-exp-seq-f1 & \seqsplit{CCTCGAGCGAGTGATTTTTTAGG} & Expression clone sequencing \\
    & 103-citrine-exp-seq-r1 & \seqsplit{GGGTTCTATCCTCCAACCTTTC} & Expression clone sequencing \\
    pMpGWB103-tdTom-NLS & 103-tdTom-exp-seq-f1 & \seqsplit{ATTTCGCTAATCATTCCCTAATTTC} & Expression clone sequencing \\
    & 103-tdTom-exp-seq-r2 & \seqsplit{CGGCCATGTTGTTGTC} & Expression clone sequencing \\
    & 103-tdTom-exp-seq-r1 & \seqsplit{AAGAAAGCTGGGTTCTATCC} & Expression clone sequencing \\
    \bottomrule
    \end{tabular}
\end{table}

\begin{figure}[htbp]
\centering
\includegraphics[width=1\textwidth]{Appendix2/Figs/1-expression-clone-pMpGWB102(35S)+citrine Map.pdf}
\caption{Plasmid map of pMpGWB102(35S)-citrine-NLS}
\label{fig:35S_citrine_map}
\end{figure}

\begin{figure}[htbp]
\centering
\includegraphics[width=1\textwidth]{Appendix2/Figs/2-expression-clone-pMpGWB102(35S)+tdTomato Map.pdf}
\caption{Plasmid map of pMpGWB102(35S)-tdTomato-NLS}
\label{fig:35S_tdTomato_map}
\end{figure}

\begin{figure}[htbp]
\centering
\includegraphics[width=1\textwidth]{Appendix2/Figs/3-expression-clone-pGWB103(EFalpha)+citrine Map.pdf}
\caption{Plasmid map of pMpGWB103(EF$\alpha$)-citrine-NLS}
\label{fig:EFalpha_citrine_map}
\end{figure}

\begin{figure}[htbp]
\centering
\includegraphics[width=1\textwidth]{Appendix2/Figs/4-expression-clone-pGWB103(EFalpha)+tdTomato Map.pdf}
\caption{Plasmid map of pMpGWB102(EF$\alpha$)-tdTomato-NLS}
\label{fig:EFalpha_tdTomato_map}
\end{figure}

\begin{table}[htbp]
\centering
\begin{tabular}{|p{6cm}|p{3cm}|p{4cm}|} % Same column widths as the second part
\hline
\textbf{Line name} &  & \textbf{Source} \\
\hline
Tak-1 & & Feng Lab \\
Tak-2 & & Feng Lab \\
Cam-2 & & Feng Lab \\
35s::tdTomato-NLS (Tak-1 male, WT female backgrounds) & & Feng Lab (JT) \\
EF$\alpha$::citrine-NLS (Tak-1 male, WT female backgrounds) & & Feng Lab (JT) \\
EF$\alpha$::tdTomato-NLS (Ta-k1 male, WT female backgrounds) & & Feng Lab (JT) \\
dn4mt1 ko \#6 & & Feng Lab \citep{RN189} \\
dn4mt1 ko \#21 & & Feng Lab \citep{RN189} \\
dnmt3b ko & & Feng Lab \citep{RN189} \\
cmta ko & & Feng Lab \citep{RN189} \\
\hline
\end{tabular}
\begin{tabular}{|p{6cm}|p{3cm}|p{4cm}|} % Now use 3 columns for the second part
\hline
\textbf{Library name} & \textbf{Library type} & \textbf{Source} \\
\hline
WT sperm & EM-seq & Feng Lab (JT) \\
WT sperm & AMD-seq & Feng Lab (JT) \\
WT sperm & AMD-seq & Feng Lab \\
dn4mt1 ko sperm & AMD-seq & Feng Lab \\
14 DAF embryo & AMD-seq & Feng Lab (JT) \\
7-8 DAF embryo & A3A-sc-BS-seq & Feng Lab (JT) \\
\hline
Archegoniophore & sRNA & \href{https://academic.oup.com/pcp/article/57/2/359/2460930}{OUP} \\
Embryo & sRNA & Feng Lab \\
Sporophyte & sRNA & Feng Lab \\
Thallus & sRNA & Feng Lab \\
\hline
Embryo & BS-seq & Feng Lab \\
Sporophyte & BS-seq & Feng Lab \\
Thallus & BS-seq & Feng Lab \\
\hline
\end{tabular}
\caption{Line names, library types, and sources.}
\end{table}


